packages/runtime/src/next/client.ts

```diff
@@ -202,6 +208,32 @@ export class Makeswift {
     return json
   }
 
+  async getPage(
+    pathname: string,
+    { locale: localeInput }: { locale?: string } = {},
+  ): Promise<GetPageResult | null> {
+    const siteVersion = MakeswiftSiteVersion.Live
+
+    const searchParams = new URLSearchParams()
+    if (localeInput) {
+      searchParams.set('locale', localeInput)
+    }
+
+    const response = await this.fetch(
+      `v1/pages/${encodeURIComponent(pathname)}/meta?${searchParams.toString()}`,
```

Comment:
For `getPageSnapshot`, we're using `v3/pages/:pathnameOrId/document`. I don't know what to call the subpath here. It's kinda like `meta`, but I'm afraid it'll confuse with the page meta and SEO. Any idea?

Reply:
Updated